tests/languages/rust/test.sh

```
@@ -0,0 +1,10 @@
+#!/bin/sh
+mkdir -p /tmp/rust-sdk
+cp -Rf /app/tests/sdks/rust/* /tmp/rust-sdk/
+cd /tmp/rust-sdk
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)